src/Providers/NovaServiceProvider.php

```diff
@@ -47,8 +48,8 @@ protected function routes(): void
      */
     protected function gate(): void
     {
-        Gate::define('viewNova', function ($user) {
-//            return in_array($user->email, []);
+        Gate::define('viewNova', function (UserModel $user) {
+            return explode('@', $user->email)[1] === env('NOVA_EMAIL_DOMAIN');
```

Comment:
Thoughts on...

Would need to add `use Illuminate\Support\Str;` to use statements. 

```suggestion
            return Str::after($user->email, '@') === env('NOVA_EMAIL_DOMAIN');
```